README.md

```diff
@@ -1,8 +1,37 @@
 # ya-httpx-client
 
-Communicate with a provider-based http server the way you communicate with any other http server
+Communicate with a provider-based HTTP server the way you communicate with any other HTTP server.
 
-## Quickstart
+## Introduction
+
+The documentation assumes reader knows what [Golem](https://www.golem.network/) is and understand basics of the 
+development of [yapapi-based services](https://handbook.golem.network/requestor-tutorials/service-development).
+
+Features:
+
+1. Deploy a HTTP server on a Golem provider in an easy way. There are no requirements on the server implementation
```

Comment:
```suggestion
1. Deploy an HTTP server on a Golem provider in an easy way. There are no requirements on the server implementation
```
It's pronounced *aitch-tee-tee-pee* so I always use 'an' before 'HTTP'. [Random guy on the internet agrees.](https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage)